core/bin/sqlite_worker.rs

```diff
@@ -214,198 +213,19 @@ async fn db_query(
     }
 }
 
-async fn delete_database_rows(
+async fn databases_delete(
```

Comment:
This is used to invalidate a database (eg after rows have been upserted into a table that is used by the DB).